app/assets/stylesheets/pageflow/themes/default/logo/variant/watermark.scss

```diff
@@ -15,6 +15,9 @@ $logo-watermark-variant-fade-in-with-header: false !default;
 /// Set opacity value of watermark logos
 $logo-watermark-variant-opacity: 0.3 !default;
 
+/// Height of the logo in phone layout.
+$logo-mobile-watermark-height: null !default;
+
```

Comment:
Nitpick: Following the naming scheme of the other variables, this should probably be called `$logo-watermark-mobile-height`.